src/ResourceManager/Consumption/Commands.Consumption.Test/ScenarioTests/UsageDetailsTests.ps1

```diff
@@ -18,274 +18,229 @@ List usage details
 #>
 function Test-ListUsageDetails
 {
-    $usageDetails = Get-AzureRmConsumptionUsageDetail -MaxCount 10
-
+    $usageDetails = Get-AzureRmConsumptionUsageDetail -Top 10
+	Assert-NotNull $usageDetails
```

Comment:
Do these tests only work on a specific subscription?

Reply:
No, we have the same internal implementation for MaxCount and Top.